docs/_posts/2019-07-14-Magic-Mirror-Project-Introduction-and-Kick-Off-Notification.md

```diff
@@ -0,0 +1,110 @@
+
+---
+title: Magic Mirror Project Introduction and Kick Off Notification
+date: 2019-07-14 20:00 +0800
+author: newip
+comments: true
+mathjax: true
+categories:
```

Comment:
`categories` will list at <https://ai-ml.club/categories/>, and will be put in the url.

So please select carefully so that the other posts can share it.

for example: `magic mirror`

Reply:
Done